src/environments/environments.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+import {Environment} from '../environment/environment';
+
+import {ReyServe} from './reyserve';
+
+class EnvironmentMap extends Map<String, {new(): Environment}> {
+}
+
+const environments = new EnvironmentMap()
```

Comment:
I'm not sure this warrants it's own file. It'll be easy to see what's going on right in polytool.ts